docs/howto/operate/new-tf-cluster.md

```diff
@@ -0,0 +1,94 @@
+Give it a descriptive name that at a glance provides context to the location and/or purpose of the cluster.
+
+The _minimum_ inputs this file requires are:
+
+- `prefix`: Prefix for all objects created by terraform.
+  Primary identifier to 'group' together resources.
+- `project_id`: GCP Project ID to create resources in.
+  Should be the id, rather than display name of the project.
+
+See the [variables file](https://github.com/2i2c-org/pilot-hubs/blob/master/terraform/variables.tf) for other inputs this file can take and their descriptions.
+
+Example `.tfvars` file:
+
+```
+prefix     = "my-awesome-project"
+project_id = "my-awesome-project-id
+```
+
+## Initialising Terraform
+
+The terraform state is located centrally in our `two-eye-two-see-org` GCP project, therefore you must authenticate `gcloud` to your `@2i2c.org` account before initialising terraform.
+
+```bash
+gcloud auth application-default login
+```
+
+Then you can change into the terraform directory and initialise
+
+```bash
+cd terraform
+terraform init
+```
+
+````{note}
+If you are deploying a cluster to a project you have access to via a different account, such as a university-affliated account, there are a few extra steps to take.
+Hopefully, this will be a rare scenario.
+
+First, you will need to provide terraform with an [access token](https://www.terraform.io/docs/language/settings/backends/gcs.html#configuration-variables) to use the state files.
+You can generate one using the below commands and logging in with your 2i2c.org account.
+
+```bash
+gcloud auth application-default login
+gcloud auth application-default print-access-token
+```
+
+Add the access token to the [terraform backend block](https://github.com/2i2c-org/pilot-hubs/blob/2ef8a4bf35bb5ee9bf04ab3db1218b8c183c5da2/terraform/main.tf#L2-L5) in `main.tf`.
+**DO NOT COMMIT THIS CHANGE.**
+Then run `terraform init` or `terraform init -reconfigure`.
+
+You can now login to your other gcloud account and proceed with the guide.
+````
+
+## Creating a new terraform workspace
+
+We use terraform workspaces so that the state of one `.tfvars` file does not influence another.
+Create a new workspace with the below command, and again give it the same name as the `.tfvars` filename.
+
+```bash
+terraform workspace new WORKSPACE_NAME
+```
+
+## Plan and Apply Changes
+
+```{note}
+Make sure the [Artifact Registry API](https://console.cloud.google.com/apis/library/artifactregistry.googleapis.com) in enabled on the project before deploying!
+```
+
+Plan your changes with the `terraform plan` command, passing the `.tfvars` file as a variable file.
+
+```bash
+terraform plan -var-file=projects/CLUSTER.tfvars
+```
+
+Check over the output of this command to ensure nothing if being created/deleted than you expected.
+You might want to ask a fellow 2i2c engineer to glance over it too to double check.
```

Comment:
I think that should be a requirement to fulfill before merging the PR... which actually led me to think this guide is not talking about the associated post-deployment PR, is that intended (to be added later)?

Reply:
I'm not sure, how does the post-deployment PR work?

Reply:
I think I was not clear enough... the process you outlined here "generates" a `.tfvars` file at a minimum. That one needs to live in a PR that eventually will be merged. I think the terraform plan output should be posted in that PR and revisited by a pair before actually deploying the cluster (maybe post-PR is not the best name). Even when there are mentions about others checking the plan, there is not an "associated" PR with the cluster generation event. 
So, I am trying to codify in the document how the process looked like for your work with the pangeo cluster and I am trying to follow the model we outlined for the org-ops in this section: https://pilot-hubs.2i2c.org/en/latest/howto/operate/setup-new-project.html#client-organization-provides-billing-account, maybe that helps with additional context.

Reply:
Ah, gotcha! Thanks for clarifying :) I'll open up another PR and try to make those two match